dcos/rpcclient.py

```diff
@@ -145,7 +144,8 @@ def http_req(self, method_fn, path, *args, **kwargs):
                 request_method=e.response.request.method,
                 request_url=e.response.request.url,
                 json_body=json_body)
-            raise DCOSException(message)
+
+            raise DCOSException(message) from e
```

Comment:
Why remove it? How do you show the `DCOSException`? I think the code that handles it could decide how to print the error.